packages/node_modules/@webex/internal-plugin-board/src/board.js

```diff
@@ -506,6 +506,22 @@ const Board = WebexPlugin.extend({
       .then((res) => res.body);
   },
 
+  /**
+   * Registers with Mercury, in a federation-aware manner
+   * @memberof Board.BoardService
+   * @param  {Object} channel - the channel to register to
+   * @returns {Promise<Board~Registration>}
+   */
+  registerToChannel(channel) {
+    return this.webex.request({
+      method: 'POST',
+      api: 'board',
+      uri: `${channel.channelUrl}/register`,
+      body: {action: 'ADD'}
+    })
+      .then((res) => res.body);
+  },
+
```

Comment:
@pcl  I wonder if there should be a corresponding method for removing the binding ? Or could clients use 'unregisterFromSharedMercury' method ?

Reply:
Yes, `unregisterFromSharedMercury()` is sufficient from what we've seen.